src/api/todos.ts

```diff
@@ -1,8 +1,36 @@
 import { Todo } from '../types/Todo';
 import { client } from '../utils/fetchClient';
 
-export const getTodos = (userId: number) => {
-  return client.get<Todo[]>(`/todos?userId=${userId}`);
+export const getTodos = async (userId: number) => {
+  const todo = await client.get<Todo[]>(`/todos?userId=${userId}`);
+
+  return todo || null;
+};
+
+type TodoData = Omit<Todo, 'id'>;
+
+export const createTodos = async (
+  {
+    userId,
+    title,
+    completed,
+  }: TodoData,
+) => {
+  return client.post<Todo[]>('/todos', {
+    userId,
+    title,
+    completed,
+  });
 };
 
-// Add more methods here
+export const deleteTodos = async (todo: Todo) => {
+  return client.delete(`/todos/${todo.id}`);
+};
+
+export const UpdateTodo = async (todo:Todo, value: boolean | string) => {
```

Comment:
```suggestion
export const updateTodo = async (todo:Todo, value: boolean | string) => {
```

Reply:
will be better to use separate arguments for completed and title values instead of using one general argument with any possible type of todo properties